go.mod

```diff
@@ -3,18 +3,15 @@ module github.com/SKF/go-enlight-sdk
 go 1.13
 
 require (
+	github.com/SKF/go-enlight-sdk/v2 v2.2.0 // indirect
 	github.com/SKF/go-eventsource v1.3.5
 	github.com/SKF/go-utility v1.8.0
-	github.com/SKF/go-utility/v2 v2.1.5
-	github.com/SKF/proto v1.27.1-go
+	github.com/SKF/go-utility/v2 v2.2.1
+	github.com/SKF/proto v2.1.0-go+incompatible
```

Comment:
This is probably causing the build issues for the v1 version. Should this really be bumped?

Reply:
Fixed